Week03/lesson06/homework/startup_matchmaker/css/style.css

```diff
@@ -0,0 +1,224 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  line-height: 1.5;
+  font-family: arial, helvetica, sans-serif;
+  font-size: 18px;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+header {
+  height: 100px;
+  background-color: #000;
+  border-bottom: 6px solid yellow;
+}
+
+h1 {
+  font-size: 30px;
+  color: #fff;
+  display: inline-block;
+  padding-top: 25px;
+}
+
+nav a {
+  display:inline-block;
+  height: 94px;
+  padding-top: 35px;
+  padding-left: 10px;
+  padding-right: 10px;
+  text-decoration: none;
+  color: #fff;
+  vertical-align: top;
+}
+
+
+nav {
+  float: right;
+  vertical-align: top;
+}
+
+nav a:hover {
+  color: yellow;
+}
+
+.banner {
+  text-align: center;
+  height: 100px;
+  background-color: #d5d9d3;
+  padding-top: 35px;
+}
+
+.heroImage {
+  height: 600px;
```

Comment:
It's probably best to let `heroImage` be as tall as the content inside of it is, so I'd remove this line and instead put `padding: 50px;` or something like that